### PR TITLE
Content sweep

### PR DESCRIPTION
**Pull request by rlewan**
### Change description

Results of a content sweep done by me, Julian, Ronan and Jackie.

### CODE REVIEW

File: src/main/app/utils/numberFormatter.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import * as numeral from 'numeral'
 
-export const NUMBER_FORMAT = '$0,0.00'
+export const NUMBER_FORMAT = '$0,0[.]00'
```
Comment (rlewan) [inline src/main/app/utils/numberFormatter.ts]: This makes the money formatter not display pennies if there are none, i.e. no `.00`.

File: src/main/features/claim/views/eligibility/government-department.njk [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+
 {% block content %}
 
   <div class="grid-row">
     <div class="column-two-thirds">
       <h1 class="heading-large">{{ t('Are you claiming against a government department?') }}</h1>
       <p>
-        {{ t('<a href="{{ governmentDepartmentUri }}" target="_blank" >See a list of government departments</a>.',
-        { governmentDepartmentUri : 'https://www.gov.uk/government/publications/serve-the-treasury-solicitor-with-legal-proceedings' }) | safe }}
+        {{ t(externalLink('See a list of government departments', governmentDepartmentUri) + '.') | safe }}
```
Comment (timja-kainos) [inline src/main/features/claim/views/eligibility/government-department.njk]: this causes the link to be translated, we should just be translating the text,
`t('See a list of government departments')`

File: src/main/features/claim/views/eligibility/index.njk [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
     <div class="column-two-thirds">
-      <h1 class="heading-large">{{ t('Find out if you can use this service') }}</h1>
+      <h1 class="heading-large">{{ t('Try the new online service') }}</h1>
+
+      <p>
+        {{ t('We’re building a new service to make it simpler, clearer and faster to make and respond to claims online.') }}
+      </p>
+
+      <p>
+        {{ t('It’s in the beta phase which means we’re making continuous improvements based on feedback from users. You can still ' + externalLink('use Money Claim Online (MCOL)', 'https://www.moneyclaim.gov.uk/web/mcol/welcome') + ' if you prefer while the new service is in beta.') | safe }}
```
Comment (timja-kainos) [inline src/main/features/claim/views/eligibility/index.njk]: the mcol link is available in config if you want to use it

File: src/main/features/response/views/confirmation.njk [hunk 4]
```diff
@@ -19,17 +19,17 @@
...
       <h2 class="bold-large form-group">{{ t('Your response') }}</h2>
 
-      <p>{{ t('We’ve emailed {{ claimant }} your defence, explaining why you reject the claim.', { claimant: claim.claimData.claimant.name  }) | safe}}</p>
+      <p>{{ t('We’ve emailed {{ claimant }} your defence, explaining why you reject the claim.', { claimant: claim.claimData.claimant.name }) | safe}}</p>
 
-      <p>{{ t('View the <a href="{{ link }}">defence against you and your response.</a>', { link: '/dashboard' }) | safe }}</p>
+      <p>{{ t('View the <a href="{{ link }}">claim against you and your response.</a>', { link: '/dashboard' }) | safe }}</p>
 
       <h2 class="heading-medium">{{ t('What happens next') }}</h2>
-      <p>{{ t('If {{ defendant }} accepts your defence the claim will be cancelled.', { defendant: claim.claimData.defendant.name }) | safe }}</p>
```
Comment (timja-kainos) [inline src/main/features/response/views/confirmation.njk]: nice

**Comment by rlewan** [comment]
Green build on dev Jenkins 😎: https://dev-build.reform.hmcts.net/blue/organizations/jenkins/hmcts%2Fcmc-citizen-frontend/detail/PR-460/17/pipeline/

**Pull request merged**